report_templates/ansible_inventory.erb

```diff
@@ -35,6 +35,14 @@ template_inputs:
   value_type: plain
   options: "yes\r\nno"
   default: yes
+- name: Host Parameters
+  required: true
+  input_type: user
+  description: Whether should be Host Parameters included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
```

Comment:
as it turned out, this needs to be wrapped in quotes `"no"`, otherwise YAML parses it as false, which is not one of "yes", "no" options.

Reply:
Thanks, done.